src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+import * as actions from './actionTypes';
+
+let lastId = 0;
+// reducer
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case actions.ADD_BOOK:
+      return [
+        ...state,
+        {
+          // eslint-disable-next-line no-plusplus
```

Comment:
- [OPTIONAL ] Kindly remove all unused code this will make your code mush noise, It is always a good practice to fix all linters errors instead of disabled it 😉 💯